client/components/app.login/view.js

```diff
@@ -0,0 +1,17 @@
+Template['login'].events({
+  "click .login-btn": function(e) {
+  	loginFn();
+  }
+});
+
+function loginFn(){
+  Meteor.loginWithGithub({
+    requestPermissions : ['user']
+  },function(err){
```

Comment:
Do we need to redirect the user to some page? 
Or may be to show some loading indicator.